search/quality_eval.go

```diff
@@ -539,3 +569,83 @@ func WriteToCsv(path string, records [][]string) error {
 	}
 	return nil
 }
+
+func GetLatestUidByFilters(filters []Filter, db *sql.DB) (string, error) {
```

Comment:
What [latest] expectations we support here?
Please add comment with description like in parse expectation function, see comment in lines 249 to 260.